File: CPP/beginner/1008.cpp
#include <stdio.h>
 
int main() {
 
   int Number = 0, Hours = 0;
   double Salary = 0, SalaryHour = 0;
   
   scanf("%d", &Number);
   scanf("%d", &Hours);
   scanf("%lf", &SalaryHour);
   
   Salary = SalaryHour * Hours;
   
   printf("NUMBER = %d\n", Number);
   printf("SALARY = U$ %.2lf\n", Salary);
 
    return 0;
}

File: CSharp/beginner/1008.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        int number = int.Parse(Console.ReadLine());
        int hoursWorked = int.Parse(Console.ReadLine());
        double hourlyRate = double.Parse(Console.ReadLine());

        double salary = hourlyRate * hoursWorked;

        Console.WriteLine($"NUMBER = {number}");
        Console.WriteLine($"SALARY = U$ {salary:F2}");
    }
}
